content/contributing/development.rst

```diff
@@ -9,3 +8,151 @@ Development
+   development/git_guidelines
+
+If you are reading this, chances are that you are interested in learning how to contribute to the
+codebase of Odoo. Whether that's the case or you landed here by accident, we've got you covered!
+
+.. seealso::
+   :doc:`Discover other ways to contribute to Odoo <../contributing>`
+
+When you feel ready, jump to the :ref:`contributing/development/setup` section to begin your journey
+in contributing to the development of Odoo.
+
+.. _contributing/development/setup:
+
+Environment setup
+=================
+
+The instructions below help you prepare your environment for making local changes to the codebase
+and then push them to GitHub. Skip this section and go to
+:ref:`contributing/development/first-contribution` if you have already completed this step.
+
+#. .. include:: create_github_account.rst
+#. .. include:: configure_github_account.rst
+#. Go to `github.com/odoo/odoo <https://github.com/odoo/odoo>`_ and click on the :guilabel:`Fork`
+   button in the top right corner to create a fork (:dfn:`your own copy`) of the repository on your
+   account. Do the same with `github.com/odoo/enterprise <https://github.com/odoo/enterprise>`_ if
+   you have access to it. This creates a copy of the codebase to which you can make changes without
+   affecting the main codebase. Skip this step if you work at Odoo.
+#. .. include:: install_git.rst
+#. .. include:: configure_git_authorship.rst
+#. :ref:`Install Odoo from the sources <setup/install/source>`. Make sure to fetch the sources
+   through Git with SSH.
+#. Configure Git to push changes to your fork(s) rather than to the main codebase. If you work at
+   Odoo, configure Git to push changes to the shared forks created on the account **odoo-dev**.
+
+   .. tabs::
+
+      .. tab:: Link Git with your fork(s)
+
+         In the command below, replace `<your_github_account>` with the name of the GitHub account
+         on which you created the fork(s).
+
+         .. code-block:: console
+
+            $ git remote add dev git@github.com:<your_github_account>/odoo.git
+
+         If you have access to `odoo/enterprise`, configure the related remote too.
+
+         .. code-block:: console
+
+            $ git remote add dev git@github.com:<your_github_account>/enterprise.git
+
+      .. tab:: Link Git with odoo-dev
+
+         .. code-block:: console
+
+            $ git remote add dev git@github.com:odoo-dev/odoo.git
+            $ git remote set-url --push origin no_push
+
+            $ git remote add dev git@github.com:odoo-dev/enterprise.git
+            $ git remote set-url --push origin no_push
+
+#. That's it! You are ready to :ref:`make your first contribution
+   <contributing/development/first-contribution>`.
+
+
+.. _contributing/development/first-contribution:
+
+Make your first contribution
+============================
+
+.. important::
+   - Developing in Odoo can be challenging for beginners. We recommend you to be knowledgeable
```

Comment:
```suggestion
   - Developing for Odoo can be challenging for beginners. We recommend you to be knowledgeable
```

Reply:
It's meant as "in the Odoo... ecosystem (?)". Do you prefer that or "Odoo development can be challenging"?

Reply:
We already use "Odoo" as different things: the company, the software, a specific database. Some clarification is welcome ^^
I wanted to tell you  "go for ecosystem," but a quick search made me realize that "Odoo ecosystem" is already used for two different things.
Damn...